zerver/actions/user_groups.py

```diff
@@ -169,11 +171,21 @@ def do_send_create_user_group_event(
     members: list[UserProfile],
     direct_subgroups: Sequence[UserGroup] = [],
 ) -> None:
+    if user_group.creator is not None:
+        creator_id = user_group.creator.id
+    else:
+        creator_id = None
```

Comment:
Same comment that you don't need this conditional, just use `user_group.creator_id` below.